docs/_getting-started-linux-android.md

```diff
@@ -47,23 +47,21 @@ Next, select the "SDK Tools" tab and check the box next to "Show Package Details
 
 Finally, click "Apply" to download and install the Android SDK and related build tools.
 
-<h4>3. Configure the ANDROID_HOME environment variable</h4>
+<h4>3. Configure the ANDROID_SDK_ROOT environment variable</h4>
 
 The React Native tools require some environment variables to be set up in order to build apps with native code.
 
 Add the following lines to your `$HOME/.bash_profile` or `$HOME/.bashrc` (if you are using `zsh` then `~/.zprofile` or `~/.zshrc`) config file:
 
 ```shell
-export ANDROID_HOME=$HOME/Android/Sdk
-export PATH=$PATH:$ANDROID_HOME/emulator
-export PATH=$PATH:$ANDROID_HOME/tools
-export PATH=$PATH:$ANDROID_HOME/tools/bin
-export PATH=$PATH:$ANDROID_HOME/platform-tools
+export ANDROID_SDK_ROOT=$HOME/Library/Android/sdk
+export PATH=$PATH:$ANDROID_SDK_ROOT/emulator
+export PATH=$PATH:$ANDROID_SDK_ROOT/platform-tools
```

Comment:
Which just those 2 you won't be able to call `sdkmanager` which is inside `tools/bin/sdkmanager`. Could you cross check?

Reply:
`tools/bin/sdkmanager` is deprecated. It is still installed because of reasons explained here: https://issuetracker.google.com/issues/67495440?pli=1#comment43

If you want to have access to the latest SDKManager CLI then you need to install the command line tools. In the comment above you have stated you do not wish to have command line tools as a hard dependency. 

I am not sure how you want to proceed. Revert to using the deprecated binary and caveat you have to use JDK 10 or below or install command line tools?

This maybe easier to jump onto a video call as at present I feel I am running in circles slightly as clearly not explaining the issue well enough.

Reply:
Ahh I see. Thanks for the issue pointer. We're safe with the paths you updated here then. If the user needs to invoke `sdkmanager`, they'll have to manually install the extra package you pointed.